Library/Formula/devd.rb

```diff
@@ -0,0 +1,132 @@
+    url "https://github.com/jessevdk/go-flags.git", :revision => "fc93116606d0a71d7e9de0ad5734fdb4b8eae834"
+  end
+
+  # devd dependencies
+  go_resource "github.com/bmatcuk/doublestar" do
+    # v1.0.1
+    url "https://github.com/bmatcuk/doublestar.git", :revision => "4f612bd6c10e2ef68e2ea50aabc50c3681bbac86"
+  end
+
+  go_resource "github.com/dustin/go-humanize" do
+    url "https://github.com/dustin/go-humanize.git", :revision => "64dbdae0d393b7d71480a6dace78456396b55286"
+  end
+
+  go_resource "github.com/fatih/color" do
+    url "https://github.com/fatih/color.git", :revision => "9aae6aaa22315390f03959adca2c4d395b02fcef"
+  end
+
+  go_resource "github.com/goji/httpauth" do
+    url "https://github.com/goji/httpauth.git", :revision => "c1b2bcd8769bd15cc56751223fd4b9f45ca987ca"
+  end
+
+  go_resource "github.com/gorilla/websocket" do
+    url "https://github.com/gorilla/websocket.git", :revision => "361d4c0ffd78338ebe0a9e6320cdbe115d7dc026"
+  end
+
+  go_resource "github.com/juju/ratelimit" do
+    url "https://github.com/juju/ratelimit.git", :revision => "772f5c38e468398c4511514f4f6aa9a4185bc0a0"
+  end
+
+  go_resource "github.com/mitchellh/go-homedir" do
+    url "https://github.com/mitchellh/go-homedir.git", :revision => "d682a8f0cf139663a984ff12528da460ca963de9"
+  end
+
+  go_resource "github.com/rjeczalik/notify" do
+    url "https://github.com/rjeczalik/notify.git", :revision => "1869adb163fffce8fb5b8755379d1042cdb4c4f8"
+  end
+
+  go_resource "github.com/toqueteos/webbrowser" do
+    # v1.0
+    url "https://github.com/toqueteos/webbrowser.git", :revision => "21fc9f95c83442fd164094666f7cb4f9fdd56cd6"
+  end
+
+  go_resource "github.com/alecthomas/template" do
+    url "https://github.com/alecthomas/template.git", :revision => "b867cc6ab45cece8143cfcc6fc9c77cf3f2c23c0"
+  end
+  go_resource "github.com/alecthomas/units" do
+    url "https://github.com/alecthomas/units.git", :revision => "2efee857e7cfd4f3d0138cc3cbb1b4966962b93a"
+  end
+  go_resource "github.com/mattn/go-colorable" do
+    url "https://github.com/mattn/go-colorable.git", :revision => "51a7e7a8b1665b25ca173debdc8d52d493348f15"
+  end
+  go_resource "github.com/mattn/go-isatty" do
+    url "https://github.com/mattn/go-isatty.git", :revision => "d6aaa2f596ae91a0a58d8e7f2c79670991468e4f"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://go.googlesource.com/crypto.git", :revision => "575fdbe86e5dd89229707ebec0575ce7d088a4a6"
+  end
+
+  go_resource "golang.org/x/net" do
+    url "https://go.googlesource.com/net.git", :revision => "c764672d0ee39ffd83cfcb375804d3181302b62b"
+  end
+
+  go_resource "gopkg.in/alecthomas/kingpin.v2" do
+    # v2.4.1
+    url "https://github.com/alecthomas/kingpin.git", :revision => "95529ad11b3c862a5b828a2142b9e50db579cf2c"
+  end
+
+  def install
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+    ENV["GOPATH"] = buildpath
+    ENV.prepend_create_path "PATH", buildpath/"bin"
```

Comment:
BTW, it doesn't work with just prepend_path:

```
HEAD is now at 95529ad Use gopkg import to avoid duplicate fetch.
==> go install github.com/GeertJohan/go.rice/rice
==> rice embed-go
Failed to execute: rice
/usr/local/Library/Homebrew/debrew.rb:11:in `raise'
BuildError: Failed executing: rice embed-go
1. raise
2. ignore
3. backtrace
4. irb
5. shell
Choose an action: 5
When you exit this shell, you will return to the menu.
bash-3.2$ echo $PATH
/usr/local/Library/ENV/4.3:/usr/local/opt/go/bin:/usr/local/opt/git/bin:/usr/bin:/bin:/usr/sbin:/sbin
bash-3.2$
```

to use ENV.prepend_path, directory specified as argument must exist.
So, there are two options:
1. move that line after rice tool installed, and change it to ENV.prepend_path
2. let ENV.prepend_create_path to create that directory and add it to PATH before rice tool installed, like it's now.